modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/NewTopicModal.es.js

```diff
@@ -39,16 +39,16 @@ export default ({
 	const [createNewTopic] = useMutation(createTopicQuery);
 
 	const isValidTopic = (topic) => {
-		const hyphens = /-+/g;
-		if (hyphens.test(topic)) {
+		const invalidCharacters = /.*[-|&|'|@|\\\\|\]|}|:|,|=|>|/|<|\n|[|{|||+|#|`|?|\\"|\r|;|/|*|~|%]/g;
```

Comment:
So many? including ???????

Reply:
I've included all the characters that we reference in the error message and I added the % because it affects the redirections. The problem with the characters in topics names is that as the name of the topic will be in the URL, some characters can be problematic. So, I've decided to include all of them but we can discuss that :)